.travis.yml

```diff
@@ -0,0 +1,19 @@
+language: dart
+
+# Re-use downloaded pub packages everywhere.
+cache:
+  directories:
+    - $HOME/.pub-cache
+
+dart:
+  # TODO: uncomment this when we have a way to only run the dartfmt check on
+  #       the stable channel of Dart
+  # - 2.3.2
```

Comment:
One possibility would be two have different scripts for each of the dart versions [like we do in react-dart](https://github.com/cleandart/react-dart/blob/master/.travis.yml).

Reply:
Yeah that might be the best option.. I get annoyed at how much duplication there is with that approach, so I was considering adding a util to dart_dev v3 like `whenDartVersionIs(constraint, tool)` that would wrap a tool and only run it when active dart version matches the constraint, but maybe that's overkill